api/src/controllers/profile.ts

```diff
@@ -83,15 +84,15 @@ export const fetchProjectProfile = async (
   const { profileId } = params;
 
   try {
-    const record = await ProfileModel.findById(Number(profileId));
+    const projectDetails = await ProfileModel.findById(Number(profileId));
 
-    const notAuthorized = isNotAuthorized(record, user);
+    const isAuthorized = getAuthorization(profileId, user, projectDetails, undefined);
 
-    if (notAuthorized) {
-      throw notAuthorized;
+    if (!(isAuthorized)) {
+      throw isAuthorized;
     }
```

Comment:
Just nitpick on readability. Currently getAuthorization either throw an Error or return true. It's just a bit odd to "throw isAuthorized" when "not authorized"?

Reply:
Absolutely agree, I have adjusted to isNotAuthorized to reflect the logic more appropriately.